src/presenter/questionsListPresenter.js

```diff
@@ -0,0 +1,48 @@
+import model from "../model/questionModel";
+
+class QuestionsListPresenter {
+    onCreateQuestion() {
+        window.location.assign("#/create-question");
+    }
+
+
+    onFilterByTag(){
+        for(var iterator = 0; iterator < model.state.questions.length; ++iterator){
+            var tagFound = false;
+            for(var tagIterator = 0; tagIterator < model.state.questions[iterator].tags.length; ++tagIterator){
+                if(model.state.questions[iterator].tags[tagIterator]  === model.state.filterTag){
+                    tagFound = true;
+                    model.state.filteredQuestions.concat(model.state.questions[iterator]);
```

Comment:
There are several problems here:

 - As above: this should be done inside the model.
 - Concat returns a new array (which you are not using, so it basically means that this line is useless).
 - You should not attempt to manipulate the model state inside the presenter directly (you should do it via model methods).
 - You can use the JS `filter` function instead of doing loops.